src/psyclone/tests/psyir/frontend/where_handler_test.py

```diff
@@ -183,6 +183,9 @@ def test_where_symbol_clash(parser):
     reader = FortranStringReader("module my_test\n"
                                  "contains\n"
                                  "subroutine widx1()\n"
+                                 "  logical, dimension(5,5,5) :: dry\n"
```

Comment:
Any particular reason for this, other than more valid code?

Reply:
Yes, just proofing against the future when the NEMO API will have a symbol table.